src/common/middlewares/game.middleware.ts

```diff
@@ -0,0 +1,20 @@
+import { StatusCodes, ReasonPhrases } from 'http-status-codes';
+import { GameService } from '../../game';
+import { Middleware } from './middleware.type';
+
+export const gameMiddleware: Middleware = async (req, res, next) => {
```

Comment:
Why using middleware? It just doesn't seem right really. @Veryyapeee let me know if this is needed. I think the cleaner solution would be checking it in request.